src/parser/html/MarkupHandler.js

```diff
@@ -380,6 +382,11 @@ module.exports = class MarkupHandler {
 
   // eslint-disable-next-line class-methods-use-this
   _lookupPlugin(name) {
-    return PLUGINS[name];
+    // eslint-disable-next-line no-prototype-builtins
```

Comment:
Re-introduced an error when a `data-sly-*` is encountered that is not supported.